### DISCUSSION TITLE
Pinocchio on a Raspberry Pi :tada:

### DISCUSSION THREAD

**Discussion by stephane-caron** [stack-of-tasks/pinocchio#1972] [Show and tell]
Today I tried ``pip install pin`` on a Raspberry Pi (``aarch64``), and it just worked out of the box :star_struck:

I don't know when it happened during Pinocchio releases, but this is amazing! Thanks to all the people involved in the process and I guess @nim65s in particular :+1:

**Comment by nim65s** [comment]
Actually, #1960 was about `aarch64` stop working, so yes, it is supposed to work :)

**Comment by nim65s** [comment]
Btw, we can easily add `ppc64le` and `s390x`, if someone needs it
